CHANGELOG.rst

```diff
@@ -1,5 +1,22 @@
 Changelog
 =========
+0.3.0
+`````
+
+Version 0.3.0 updates python-bsonjs's vendored copy of libbson to 1.20.0
+http://mongoc.org/libbson/1.20.0/
+https://github.com/mongodb/mongo-c-driver/blob/1.20.0/src/libbson/NEWS
+
+This release also provides a number of improvements and features:
+
+- Added testing and support for Windows
+- Added support for Python 3.7, 3.8, 3.9, and 3.10
+- Added a new keyword argument to the `dumps` and `dump` functions: `mode`. It
+  can be one of: bsonjs.LEGACY, bsonjs.CANONICAL, or bsonjs.RELAXED. Which are:
+  libbson's legacy extended JSON format, MongoDB Extended JSON 2.0 canonical
+  mode, and MongoDB Extended JSON legacy mode.
```

Comment:
This sentence repeats "legacy" twice. I think the second is supposed to be "relaxed".

Reply:
Done.